react/Header/NotificationsLink/NotificationsLink.js

```diff
@@ -0,0 +1,54 @@
+import React, { Fragment } from 'react';
+import BellIcon from '../../BellIcon/BellIcon';
+import styles from './NotificationsLink.less';
+import { getCookie } from './utils';
+import { AUTHENTICATED } from '../../private/authStatusTypes';
+
+class NotificationsLink extends React.Component {
+  state = {
+    isInExperiment: false
+  }
+
+  componentDidMount() {
+    const EXPERIMENT_ID = '58CC4E12-A4E0-4C0B-BD2A-681FD98FDC2E';
+    const visitorId = getCookie('JobseekerVisitorId');
```

Comment:
Can we move the `getCookie` call into the try/catch just in case?

Reply:
Done